docs/pages/components/Code.mdx

```diff
@@ -82,6 +94,22 @@ By default, Code components will wrap code forcing breaks where it can. If the o
       type: 'boolean',
       default: 'false',
       description: 'Preserve the original markup, forcing the component to scroll when it overflows.'
+    },
+    {
+      name: 'label',
+      type: 'React.ContentNode',
+      description: 'Label for the copy button'
```

Comment:
This is actually the label for the heading and not the button. The button's name gets set via `CopyButtonProps`

```suggestion
      description: 'Label for the code heading'
```